flask_hal/link.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""
+flask_hal.link
+==============
+
+Implements the ``HAL`` Link specification.
+"""
+
+# Standard Libs
+import json
+
+
+VALID_LINK_ATTRS = [
+    'name',
+    'title',
+    'type',
+    'deprecation',
+    'profile',
+    'templated',
+    'hreflang'
+]
+
+
+class Links(object):
+    """Build a collection of ``HAL`` link objects.
+
+    Example:
+        >>> from flask_hal.link import Link, Links
+        >>> l = Links(
+        ...     Link('foo', 'http://foo.com'),
```

Comment:
Just a thought but wondering whether you need to specify Link here - Links vs Link seems confusing(and you have to import both). If the Links object always contains links can you not just insert a simple key:value rather than have to specify Link(x,x) each time?